apps/constants/base.py

```diff
@@ -5,20 +5,19 @@
 
 
 # Add-on and File statuses.
-STATUS_NULL = 0
-STATUS_UNREVIEWED = 1
-STATUS_PENDING = 2
-STATUS_NOMINATED = 3
-STATUS_PUBLIC = 4
-STATUS_DISABLED = 5
+STATUS_NULL = 0  # No review type chosen yet, add-on is incomplete.
+STATUS_UNREVIEWED = 1  # Waiting for prelim review.
+STATUS_PENDING = 2  # Personas (lightweight themes) waiting for review.
```

Comment:
I don't think so (but I could be wrong), it seems to be used for personas: https://github.com/mozilla/olympia/blob/master/apps/addons/forms.py#L510